testsuite/tests/parsetree/source_jane_street.ml

```diff
@@ -1068,8 +1068,8 @@ type 'a contended_with_int : immutable_data with 'a @@ contended with int
 [%%expect{|
 type 'a list : immutable_data with 'a
 type ('a, 'b) either : immutable_data with 'a * 'b
-type 'a contended : immutable_data with 'a @@ contended
-type 'a contended_with_int : immutable_data with 'a @@ contended with int
+type 'a contended : immutable_data with 'a
```

Comment:
This looks wrong -- I think we need the `@@ contended` to remain.

Reply:
Yes I agree

Reply:
I just fixed this in @glittershark 's normalization pr. The bugfix should also resolve the other modality printing issues you commented on once it gets merged into this branch.